src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props

```diff
@@ -4,5 +4,6 @@
 		<_JavaInteropReferences>Java.Interop;System.Runtime</_JavaInteropReferences>
 		<DependsOnSystemRuntime Condition=" '$(DependsOnSystemRuntime)' == '' ">true</DependsOnSystemRuntime>
 		<CopyNuGetImplementations Condition=" '$(CopyNuGetImplementations)' == ''">true</CopyNuGetImplementations>
+		<_YieldDuringExecution Condition="'$(_YieldDuringExecution)' == ''">true</_YieldDuringExecution>
```

Comment:
Unfortunately, `$(YieldDuringToolExecution)` isn't in `xbuild`, but we can split the difference:

```xml
<_YieldDuringExecution>$(YieldDuringToolExecution)</_YieldDuringExecution>
<_YieldDuringExecution Condition=" '$(_YieldDuringExecution)' == '' ">true</_YieldDuringExecution>
```

This has the added benefit of not touching the rest of the PR. :-)